test/e2e/runner/load.go

```diff
@@ -54,9 +54,19 @@ func Load(ctx context.Context, testnet *e2e.Testnet) error {
 		case numSeen := <-chSuccess:
 			success += numSeen
 		case <-ctx.Done():
-			if success == 0 {
+			// if we couldn't submit any transactions,
+			// that's probably a problem and the test
+			// should error; however, for very short tests
+			// we shouldn't abort.
+			if success == 0 && time.Since(started) < 2*time.Second {
```

Comment:
Is the sense of the comparison reversed here? This reports an error if we had no success and the time _is_ short, but the comment suggests we'd want to accept it?

Reply:
classic @tychoish mistake! Thanks!